### ISSUE TITLE
PITCH, ROLL, HEADING, ALTITUDE OG1 vocabs missing definition 

### ISSUE THREAD

**Issue by emmerbodc** [OceanGlidersCommunity/OG-format-user-manual#272]
moderator: @OceanGlidersCommunity/format-maintainers
Hello @danibodc 

It's been spotted that three of our terms in the OG1 vocab do not have definitions. 
https://vocab.nerc.ac.uk/collection/OG1/current/PITCH/
https://vocab.nerc.ac.uk/collection/OG1/current/ALTITUDE/
https://vocab.nerc.ac.uk/collection/OG1/current/HEADING/
https://vocab.nerc.ac.uk/collection/OG1/current/ROLL/

Can these be added once agreement on the definitions please 

@callumrollo @fmargirier @vturpin @jenseva  @danibodc are you happy with these definitions below? I've copied the concept name.  

PITCH 
Orientation (pitch) of measurement platform by inclinometer
ROLL
Orientation (roll angle) of measurement platform by inclinometer
HEADING
Orientation (horizontal relative to magnetic north) of measurement platform {heading} by compass
ALTITUDE
Height above bed in the water body

Thanks,
Emma

**Comment by jenseva** [comment]
Thanks Emma! Here's some potential rewording:

PITCH
Orientation (pitch angle in degrees) of measurement platform as typically measured by an inclinometer
ROLL
Orientation (roll angle in degrees) of measurement platform as typically measured by an inclinometer
HEADING
Orientation (horizontal relative to magnetic north) of measurement platform {heading} as typically measured by a compass
ALTITUDE
Height (meters) of measurement platform above bed in the water body as typically measured by an altimeter

Also, re: the heading wording, some data flows may have the true north correction fairly early on in the processing. Do you want separate variable for true vs magnetic?

**Comment by fmargirier** [comment]
Looking good !
Should we also specify a unit for the heading in degrees ?

for the true vs magnetic, as long as we choose one or the other it should be fine no ?

**Comment by callumrollo** [comment]
A more general questions, should these definitions have units as @jenseva recommends? iirc there was some discussion before as to whether this is within the purview of OG1, or should be left out of this vocabulary for specification by the `units` attribute and/or use of e.g. P06

**Comment by danibodc** [comment]
Hello all, thanks for the feedback! For the heading I would recommend creating separate OG1 terms for true vs magnetic so they can be distinguished, I can create this if required?

R.e. units, we do map OG1 terms to a P06 term, but we encourage users to choose a P06 that is suitable for their dataset. Users shouldn't limit themselves to whatever P06 we have mapped to the OG1 term. 

I'm happy with the definitions above and can update the terms if everyone is happy with the above.

**Comment by danibodc** [comment]
Yes I'd agree with you Emma r.e. the 'typically measured by' statement, as the term is specific to the method in this case inclinometer. The statement would be more suited to a more generic pref label that doesn't mention the method.

**Comment by danibodc** [comment]
Happy New Year all, just coming back to this.

R.e. the prefLabel for HEADING in OG1: 
Orientation (horizontal relative to **magnetic north**) of **measurement platform** {heading} by compass

I will need to update the P01 mapping. Currently it is mapped to https://vocab.nerc.ac.uk/collection/P01/current/HEADCM01/: Orientation (horizontal) of **measurement device** relative to **True North** {heading}

Which refers to 'measurement device' and 'True North' as opposed to 'measurement platform' and 'magnetic north'. Therefore I will map OG1.HEADING to P01.HEADCMMG: https://vocab.nerc.ac.uk/collection/P01/current/HEADCMMG/

I will then create a new OG1 term for heading relative to True North, e.g.
Unique Identifier: HEADINGTN
PrefLabel: Orientation (horizontal relative to magnetic north) of measurement platform {heading}

- Please could I have some feedback on whether prefLabel should also include **by compass**?

And will need to create a new P01 term to map this to, e.g.
Orientation (horizontal) of measurement platform relative to True North {heading}

Proposed definition for HEADING:
The rotation of a platform about its vertical/Z axis relative to magnetic north. Often referred to as yaw or bearing.

Proposed definition for HEADINGTN:
The rotation of a platform about its vertical/Z axis relative to True North. Often referred to as yaw or bearing.

---

OG1.ROLL: https://vocab.nerc.ac.uk/collection/OG1/current/ROLL/
Orientation (roll angle) of measurement platform by inclinometer

The P01 mapping will need to be updated from ROLLFG01to ROLLEI01 to reference 'inclinometer' rather than 'triaxial fluxgate compass': see http://vocab.nerc.ac.uk/collection/P01/current/ROLLEI01

Proposed OG1.ROLL definition:
The tilting rotation of a platform about its longitudinal/X axis as measured by an inclinometer.

---

OG1.PITCH: https://vocab.nerc.ac.uk/collection/OG1/current/PITCH/
Orientation (pitch) of measurement platform by inclinometer

Again the P01 mapping will need to be updated from PTCHFG01 to PTCHEI01, to reference 'inclinometer' rather than 'triaxial fluxgate compass': see http://vocab.nerc.ac.uk/collection/P01/current/PTCHEI01

Proposed OG1.PITCH definition:
The up/down rotation of a platform about its transverse/Y axis as measured by an inclinometer.

---

OG1.ALTITUDE: https://vocab.nerc.ac.uk/collection/OG1/current/ALTITUDE/
Height above bed in the water body

Proposed definition:
The vertical distance of a platform above the bed surface of a water body.

---

If everyone is happy with the above I will add the new term and update the existing mappings and definitions.
Many thanks!

**Comment by emmerbodc** [comment]
Following meeting proposed suggestions are 

edit to OG1:HEADING preflabel to remove by compass:
Orientation (horizontal relative to magnetic north) of measurement platform {heading} 

P01 linkage to be changed. 

definition adding: The rotation of a platform about its upward axis relative to magnetic north. Often referred to as yaw or bearing

new term HEADINGTN
PrefLabel: Orientation (horizontal relative to magnetic north) of measurement platform {heading}

P01 linkage  P01.HEADCMMG: https://vocab.nerc.ac.uk/collection/P01/current/HEADCMMG/

definition adding: The rotation of a platform about its upward axis relative to True North. Often referred to as yaw or bearing

--

OG1.ROLL: https://vocab.nerc.ac.uk/collection/OG1/current/ROLL/
Orientation (roll angle) of measurement platform by inclinometer

The P01 mapping will need to be updated from ROLLFG01to ROLLEI01 to reference 'inclinometer' rather than 'triaxial fluxgate compass': see http://vocab.nerc.ac.uk/collection/P01/current/ROLLEI01

Proposed OG1.ROLL definition:
The tilting rotation of a platform as measured by an inclinometer.

--

OG1.PITCH: https://vocab.nerc.ac.uk/collection/OG1/current/PITCH/
Orientation (pitch) of measurement platform by inclinometer

Again the P01 mapping will need to be updated from PTCHFG01 to PTCHEI01, to reference 'inclinometer' rather than 'triaxial fluxgate compass': see http://vocab.nerc.ac.uk/collection/P01/current/PTCHEI01

Proposed OG1.PITCH definition:
The up/down rotation of a platform as measured by an inclinometer.


--

OG1.ALTITUDE: https://vocab.nerc.ac.uk/collection/OG1/current/ALTITUDE/
Height above bed in the water body

Happy with P01 mapping.

Proposed definition:
The vertical distance of a platform above the bed surface of a water body.

--
Note  - agreed to keep instrumentation in description to allow people to link the same measurement from different instrument types. 


@callumrollo @fmargirier @danibodc @jenseva  do you agree this these to be published?

**Comment by fmargirier** [comment]
sounds good to me ! Thanks for the work !
